src/utils/logging.cxx

```diff
@@ -35,6 +35,8 @@
 
 #if defined(__linux__) || defined(__MACH__)
 char logbuffer[4096];
+#elif defined(ESP32)
+char logbuffer[2048];
```

Comment:
256 was not enough for your log lines?

Reply:
For the CS it wasn't. Perhaps this could be smaller 1k instead of 2k though.

Reply:
dropped to 1k